defaults/main.yml

```diff
@@ -1456,6 +1456,26 @@ owncloud__nginx_access_log_assets: True
 # TODO: Confirm that this variable does what it says.
 owncloud__nginx_client_body_temp_path: ''
 
+owncloud__nginx__php_status: False
+owncloud__nginx__php_status_location:
+  pattern: '~ ^/(php_status|php_ping)$'
+  options: |
+    access_log off;
+    {% if nginx_status_localhost %}
+    {% for address in nginx_status_localhost | unique %}
+    allow {{ address }};
+    {% endfor %}
+    {% endif %}
+    {% if nginx_status %}
+    {% for address in nginx_status | unique %}
+     allow {{ address }};
+    {% endfor %}
+    {% endif %}
+    deny all;
+    include fastcgi_params;
+    fastcgi_index index.php;
+    fastcgi_param SCRIPT_FILENAME $document_root$fastcgi_script_name;
+    fastcgi_pass php_owncloud;
```

Comment:
In my opinion, the best way to solve this would be to switch the `debops.owncloud` role to use the `php` type nginx configuration and fix any issues there if they arise. This way the role should be ready to be used with Apache role if/when it will be created.

To make the role usable this way, you shouldn't use the variables from other roles directly. If there are any variables in `debops.nginx` that would be useful in different roles, they should be exposed via Ansible local facts.